test/e2e/bootstrap/OWNERS

```diff
@@ -0,0 +1,14 @@
+approvers:
```

Comment:
What should we have here @kubernetes/sig-cluster-lifecycle-pr-reviews?
Who will/can sign up for maintaining this e2e?
I think we can/should have at least @mikedanese @jbeda and myself, any other volunteers?